src/test/java/entities/DayTest.java

```diff
@@ -0,0 +1,17 @@
+package entities;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DayTest {
+
+    @Test
+    public void testEmptyConstructor(){
+        Day d = new Day(new HashMap<>(), new HashMap<>());
+        assertEquals(new HashMap<>(), d.getEvents());
+        assertEquals(new HashMap<>(), d.getTasks());
+    }
+}
```

Comment:
Would benefit from more thorough testing once Event and Task classes are created, but with the available code at time of writing the testing is as extensive as possible.

Reply:
This naming convention is not consistent with your WeekTest.java test name